includes/ajax-handlers/class-wc-stripe-checkout-sessions-ajax-handler.php

```
@@ -70,38 +71,100 @@ public function create_checkout_session(): void {
+				throw new Exception( __( 'Checkout session ID is required.', 'woocommerce-gateway-stripe' ) );
+			}
+
+			if ( ! WC()->cart || WC()->cart->is_empty() ) {
+				throw new Exception( __( 'Your cart is currently empty.', 'woocommerce-gateway-stripe' ) );
+			}
+
+			// Recalculate totals.
+			WC()->cart->calculate_totals();
+
+			$currency   = get_woocommerce_currency();
+			$cart_total = WC_Stripe_Helper::get_stripe_amount( (float) WC()->cart->get_total( 'edit' ), $currency );
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Helper' in method 'update_checkout_session'. (undefined)

(StaticAccess)